releasenotes/notes/13.0.0-8158f0be466af558.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - OSA Variable ssh_delay is now overriden to 60 from the osa default of 5.
```

Comment:
Minor nits:

* `s/overriden/overridden/`
* Should we change `OSA` and `osa` to `OpenStack-Ansible for clarity?